impl/src/main/java/io/jsonwebtoken/impl/io/RuntimeClasspathDeserializerLocator.java

```diff
@@ -34,6 +34,8 @@
             return Classes.newInstance("io.jsonwebtoken.io.JacksonDeserializer");
         } else if (isAvailable("io.jsonwebtoken.io.OrgJsonDeserializer")) {
             return Classes.newInstance("io.jsonwebtoken.io.OrgJsonDeserializer");
+        } else if (isAvailable("com.google.gson.GsonBuilder")) {
```

Comment:
This should be checking for "io.jsonwebtoken.io.GsonDeserializer" not "com.google.gson.GsonBuilder"

Reply:
ok fixed.